app/views/shared/views/_ownership.html.haml

```diff
@@ -17,7 +17,7 @@
 %h3
   = _('Changes')
 
-= react('SetServiceOwnershipForm', { :ownerOptions => user_opts, :groupOptions => groups_opts, :ownershipIds => @ownershipitems.collect(&:id).map(&:to_s) })
+= react('SetOwnershipForm', { :ownerOptions => user_opts, :groupOptions => groups_opts, :ownershipIds => @ownershipitems.collect(&:id).map(&:to_s) })
```

Comment:
It seems that the `group_opts` and `user_opts` variables are no longer used. Can you please delete them from above?